src/reducers/org.js

```diff
@@ -1167,9 +1167,7 @@ export const setSearchFilterInformation = (state, action) => {
     const headers = files.map((file) => file.get('headers'));
 
     // show clocked times & sum if there is a clock search term
-    const clockFilters = searchFilterExpr
-      .filter((f) => f.type === 'field')
-      .filter((f) => f.field.type === 'clock');
+    // TODO these depend on clockFilters which is now in isMatch() ...
```

Comment:
It will take me a time, too, to understand all those filters again. I moved the clock filter added by you, @tarnung, to `isMatch` – but I don't understand yet what your other lines do and how we can refactor that...

It's late ^^ I just wanted to make a note for now.